### PR TITLE
Danfo: Version 1.000 added

### PR DESCRIPTION
**Pull request by emmamarichal**
d2b9348: [gftools-packager] Danfo: Version 1.000 added

* Danfo Version 1.000 taken from the upstream repo https://github.com/Afrotype/danfo at commit https://github.com/Afrotype/danfo/commit/a66fc9ded8f42ad2d39b91c9cd8a1960737ad02a.

 614b311: description updated

- [x] tags - form sent
- [ ] images
- [x] designer profiles
- [x] review

### CODE REVIEW

**Comment by felipesanches** [comment]
Please re-run the CI job for this PR because the fontbakery report was broken. I've just released a new version of Fontbakery (v0.12.0a4) that fixes it.

**Comment by m4rc1e** [comment]
@felipesanches looks ace mate. Thank you for fixing.

**Comment by emmamarichal** [comment]
@felipesanches do you know why it's not sorted by FAILS/WARN etc?

**Comment by emmamarichal** [comment]
@m4rc1e Jaro still needs designer profile, description update, but this one should be ready. The images will arrive soon, you can review and merge it.

File: ofl/danfo/DESCRIPTION.en_us.html [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<p>Danfo is an ELSH axis variable font with three main styles at 0,50, and 100. Danfo covers all of the Google SSA character set. The three styles reflect different cut-out shapes inspired by the vinyl cut-out lettering of the public transportation buses in Lagos, Nigeria.</p>
```
Comment (m4rc1e) [inline ofl/danfo/DESCRIPTION.en_us.html:1]: Nit. I don't think most users would know what the abbreviations ELSH and SSA mean. Maybe nicer to em

**Comment by m4rc1e** [comment]
Thank you. In a follow up pr, maybe address the abbreviations as well.

**Pull request merged**